app/controllers/concerns/api_token_authenticatable.rb

```diff
@@ -26,6 +27,14 @@ def current_user
 
 private
 
+  def set_custom_paper_trail_whodunnit
+    PaperTrail.request.whodunnit = if current_user.is_a?(ApplicationRecord)
+                                     current_user.to_global_id.to_s
```

Comment:
Do we set it to a global id anywhere else, or just in the API?

Reply:
just api at the moment. I can make this change in the "normal" app in another PR once i've finished this

Reply:
This will now go into another PR